### PR TITLE
Add post hadoop1

### PR DESCRIPTION
**Pull request by RackerKate**
@cyrichardson Please review.

### CODE REVIEW

**Comment by cyrichardson** [comment]
Got it, @RackerKate!

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-833b3ecf20/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-08ce1d0bfd/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-bb0ec162f4/).

**Comment by cyrichardson** [comment]
@RackerKate Suggested changes here: 
[blog_Hadoop ecosystem basics- Part 1_cyrEdits.docx](https://github.com/rackerlabs/docs-developer-blog/files/2094978/blog_Hadoop.ecosystem.basics-.Part.1_cyrEdits.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-bb0ec162f4/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-876ef49a48/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-86af248dd4/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-86af248dd4/).

**Comment by RackerKate** [comment]
@catlook For your review.

File: _posts/2018-06-12-hadoop-eco-basics-1.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+Oozie also makes it very easy to re-run failed workflows.
+
+Oozie is scalable and can manage timely execution of thousands of workflows
+(each consisting of dozens of jobs) in a Hadoop cluster.
+
+#### Scripting
+
+Developers can use Apache Pig for scripting in Hadoop. Scripting uses a
+SQL-based language and an execution environment for creating complex MapReduce
+transformations. While Pig is written in the Pig Latin "language," it's
```
Comment (catlook) [inline _posts/2018-06-12-hadoop-eco-basics-1.md]: ... written in the Pig Latin coding language, it's ...

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-1c88f7b564/)

**Pull request merged**